### PR TITLE
Added sections for qr codes

### PR DESCRIPTION
**Pull request by craftamap**

### CODE REVIEW

**Review by diehmlukas — APPROVED** [review]

File: chapters/methodology.tex [hunk 1]
```diff
@@ -34,6 +35,13 @@ \subsection{User Journey} \label{sec:user_journey}
 
 Following this process, a set of six questions querying current circumstances are posed to the user. Once all questions have been answered, the user will be thanked for their participation and presented with a percentage indicating how likely they are genuinely happy.
 
+A study may also provide an individual configuration for the app. This configuration
+can change the behavior of the app, like various timings or different implementations
```
Comment (AnnaGsell) [inline chapters/methodology.tex]: BE: behaviour

File: chapters/methodology.tex [hunk 2]
```diff
@@ -34,6 +35,13 @@ \subsection{User Journey} \label{sec:user_journey}
 
 Following this process, a set of six questions querying current circumstances are posed to the user. Once all questions have been answered, the user will be thanked for their participation and presented with a percentage indicating how likely they are genuinely happy.
 
+A study may also provide an individual configuration for the app. This configuration
+can change the behavior of the app, like various timings or different implementations
+of face/smile detectors.
+As a first step after installing soHappy, the user may scan an QR code provided
```
Comment (limkokyew) [inline chapters/methodology.tex]: "a" instead of "an"

**Review by limkokyew — APPROVED** [review]
One minor thing, but otherwise, good work!

File: chapters/methodology.tex [hunk 3]
```diff
@@ -34,6 +35,13 @@ \subsection{User Journey} \label{sec:user_journey}
 
 Following this process, a set of six questions querying current circumstances are posed to the user. Once all questions have been answered, the user will be thanked for their participation and presented with a percentage indicating how likely they are genuinely happy.
 
+A study may also provide an individual configuration for the app. This configuration
+can change the behavior of the app, like various timings or different implementations
+of face/smile detectors.
+As a first step after installing soHappy, the user may scan an QR code provided
+by the study with an installed QR-Code app. The user will be asked to open the
```
Comment (AnnaGsell) [inline chapters/methodology.tex]: maybe use 'the participant' instead of the user instead of repetition

**Review by AnnaGsell — CHANGES_REQUESTED** [review]

**Review by AnnaGsell — APPROVED** [review]

**Review by mac641 — APPROVED** [review]

**Pull request merged**